iml-services/iml-ntp/cargo.toml

```diff
@@ -0,0 +1,32 @@
+[package]
+name = "iml-ntp"
+version = "0.1.0"
+authors = ["IML Team <iml@whamcloud.com>"]
+edition = "2018"
+
+[dependencies]
```

Comment:
Audit the deps / dev-deps, I don't think you're using many of these.

Reply:
I went through each of the items and reduced it to:

```
[package]
name = "iml-ntp"
version = "0.1.0"
authors = ["IML Team <iml@whamcloud.com>"]
edition = "2018"

[dependencies]
tokio = "0.2.0-alpha.4"
tokio-postgres = "0.4.0-rc.3"
iml-wire-types = { path = "../../iml-wire-types", version = "0.2" }
iml-rabbit = { path = "../../iml-rabbit", version = "0.1.0" }
iml-postgres = { path = "../../iml-postgres", version = "0.1.0" }
iml-service-queue = { path = "../iml-service-queue", version = "0.1.0" }
futures-preview = { version = "0.3.0-alpha.18", features = ["compat"] }
tracing = "0.1.9"
tracing-subscriber = "0.1.3"
```